### PR TITLE
Clarifying about API key type for mobile SDK

### PR DESCRIPTION
**Pull request by dgtv**

### CODE REVIEW

File: source/ios_sdk/getting_started.rst [hunk 1]
```diff
@@ -39,14 +39,15 @@ Configuration
...
   .. code-block:: objc
 
-    [[Talkable manager] setApiKey:@"YOUR_TALKABLE_API_KEY" andSiteSlug:@"YOUR_SITE_SLUG"];
+    [[Talkable manager] setApiKey:@"YOUR_TALKABLE_PUBLIC_API_KEY" andSiteSlug:@"YOUR_SITE_SLUG"];
 
   .. note::
 
     You can locate your credentials inside Talkable site:
 
     - Visit https://www.talkable.com/account/sites to find you site slug
```
Comment (zhuravel) [inline source/ios_sdk/getting_started.rst]: Please fix typo here: "find you site slug" → "find **your** site slug"

File: source/android_sdk/getting_started.rst [hunk 2]
```diff
@@ -48,7 +48,8 @@ After this, add dependencies to `build.gradle`
 
     - Visit https://www.talkable.com/account/sites to find you site slug
```
Comment (zhuravel) [inline source/android_sdk/getting_started.rst]: Same here.

File: source/ios_sdk/getting_started.rst [hunk 3]
```diff
@@ -39,14 +39,15 @@ Configuration
...
+    [[Talkable manager] setApiKey:@"YOUR_TALKABLE_PUBLIC_API_KEY" andSiteSlug:@"YOUR_SITE_SLUG"];
 
   .. note::
 
     You can locate your credentials inside Talkable site:
 
-    - Visit https://www.talkable.com/account/sites to find you site slug
-    - Select site and go to **Dashboard** |rarr| **Site Settings**. Find **Integration settings** section and there you will see your API Keys. Use only the public key in your application submitted to the App Store.
+    - Visit https://www.talkable.com/account/sites to find your site slug
+    - Select site and go to **Dashboard** |rarr| **Site Settings**. Find **Integration settings** section and there you will see your API Keys.
```
Comment (vitalikdanchenko) [inline source/ios_sdk/getting_started.rst]:
```
**Dashboard** |rarr| **Settings** |rarr| **Site Settings**
```
and the same above

File: source/android_sdk/getting_started.rst [hunk 4]
```diff
@@ -39,16 +39,17 @@ After this, add dependencies to `build.gradle`
...
-      <meta-data android:name="TalkableApiKey" android:value="{{YOUR_TALKABLE_API_KEY}}" />
+      <meta-data android:name="TalkableApiKey" android:value="{{YOUR_TALKABLE_PUBLIC_API_KEY}}" />
       <meta-data android:name="TalkableSiteSlug" android:value="{{YOUR_SITE_SLUG}}" />
 
   .. note::
 
     You can locate your credentials inside Talkable site:
 
-    - Visit https://www.talkable.com/account/sites to find you site slug
+    - Visit https://www.talkable.com/account/sites to find your site slug
```
Comment (vitalikdanchenko) [inline source/android_sdk/getting_started.rst]: `www.talkable.com` -> `admin.talkable.com`

Comment (dgtv) [inline source/android_sdk/getting_started.rst]: There are a lot of places with such path. So I'll create another PR for this fix.

Comment (vitalikdanchenko) [inline source/android_sdk/getting_started.rst]: okay

**Pull request merged**